cfme/tests/cloud/test_provisioning.py

```diff
@@ -153,3 +165,72 @@ def _finish_method():
             soft_assert(provider_mgmt.volume_attachments(volume)[vm_name] == device_mapping)
         instance.delete_from_provider()  # To make it possible to delete the volume
         wait_for(lambda: not instance.does_vm_exist_on_provider(), num_sec=180, delay=5)
+
+
+# Not collected for EC2 in generate_tests above
+@pytest.mark.meta(blockers=[1160342])
+def test_provision_with_boot_volume(
+        request, setup_provider, provider_crud, provisioning, vm_name, provider_mgmt, soft_assert,
+        provider_type, default_domain_enabled):
+    """ Tests provisioning from a template and attaching one booting volume.
+
+    Metadata:
+        test_flag: provision
```

Comment:
Should probably add a volumes flag too...remember to add it in the yamls also in the test_flags value.